### DISCUSSION TITLE
Generating a pivot table based on a table that does not start on the 1st row

### DISCUSSION THREAD

**Discussion by LS01-Personal** [dfinke/ImportExcel#1397] [Q&A]
I'm trying to generate a pivot table based on a table in another sheet.  This source table does not start on the 1st row since there are a few header rows above it.

How do I tell Export-Excel where to find the starting row?  I assume it must be the PivotTableDefinition hash.

Thanks.

**Comment by dfinke** [comment]
please post a repro script and the xlsx
